master/getting-started/mesos/installation/dc-os/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Installing Calico in DC/OS
+---
+
+The following information details Calico's installation and runtime dependencies
+in DC/OS, and looks at how to leverage Calico-DC/OS Framework to get up and running.
+
+## Background
+
+Calico provides multi-host networking for DC/OS, giving each task it's own IP
+address and isolated networking namespace, with highly configurable policy.
```

Comment:
suggest "highly flexible" instead of "configurable".  (or "highly flexible policy configuration").